src/meielement.cpp

```diff
@@ -98,23 +98,23 @@ void mei::MeiElement::generateAndSetId() {
     this->setId(out);
 }
 
-const string mei::MeiElement::getId() {
+const string mei::MeiElement::getId() const {
```

Comment:
Here and in other methods: Returning a string by value as const doesn't make a lot of sense. Simply returning `string` should be preferred.

Reply:
Perhaps it was originally meant to return by reference? Anyway, I'd leave this for now.